dask/dataframe/io/tests/test_parquet.py

```diff
@@ -922,10 +919,20 @@ def test_read_parquet_custom_columns(tmpdir, engine):
     ],
 )
 def test_roundtrip(tmpdir, df, write_kwargs, read_kwargs, engine):
+    if "x" in df and df.x.dtype == "M8[ns]" and "arrow" in engine:
+        pytest.xfail(reason="Parquet pyarrow v1 doesn't support nanosecond precision")
     if (
-        PANDAS_GT_130
+        "x" in df
+        and df.x.dtype == "M8[ns]"
         and engine == "fastparquet"
+        and fastparquet.__version__ <= LooseVersion("0.6.3")
```

Comment:
Should we wait for >0.6.3 to drop so CI is actually covering the changes here?

Reply:
If needed, I'll just fix things when the time comes. Otherwise, I'd need to come back here and change this anyway.

Reply:
xref https://github.com/dask/dask/pull/7884 for including the dev version of `fastparquet` in our upstream CI build